src/ModelTable/ModelTable.tsx

```diff
@@ -81,7 +81,9 @@ export const ModelTable = Object.assign(
                 typeof scrollable === 'object' ? true : scrollable ?? true
               }
               className={
-                typeof scrollable === 'object' ? scrollable?.className : ''
+                typeof scrollable === 'object'
+                  ? `${scrollable?.className} h-full`
+                  : 'h-full'
```

Comment:
do `cn(typeof scrollable === 'object' && scrollable.className, 'h-full')`

Can you fix this to use `cn` above for `BorderWrapper` as well

Reply:
Also I just checked and seems like `h-full` is already present on ScrollArea

Reply:
nvm i made the fixes and remved h-full